ci/requirements-py27-min.yml

```diff
@@ -1,9 +1,12 @@
 name: test_env
+channels:
+  - conda-forge
 dependencies:
   - python=2.7
   - pytest
   - numpy==1.9.3
   - pandas==0.15.0
+  - rasterio
```

Comment:
I think we can roll back these changes.  This test setup is just for required dependencies which `rasterio` will not be.